concourse/scripts/pipelines-bosh-cloudfoundry.sh

```diff
@@ -54,6 +55,14 @@ prepare_environment() {
 
   download_git_id_rsa
   get_git_concourse_pool_clone_full_url_ssh
+
+  if [ "${ENABLE_DATADOG:-}" = "true" ]; then
+    export PASSWORD_STORE_DIR=~/.paas-pass
```

Comment:
Do we have a list of env vars developers are expected to set or to not set as part of their development environment?
Is this something an individual dev might override?
If so, should we check whether it is already set before overriding it?